docs/introduction.md

```diff
@@ -0,0 +1,19 @@
+The Kaltura Player integrates different packages into one single package, creating a powerful media player.
```

Comment:
The Kaltura Player takes on a highly modular approach for creating a powerful media player.
Each functionality of the player is isolated to separate package focused on delivering specific set of abilities.
This approach enables high Modularity, Extensibility, Simplicity and Maintainability.

@AnnaAleksandrowicz - WDYT?

Reply:
The Kaltura Player utilizes a highly modular approach for creating a powerful media player.
Each functionality of the player is isolated into separate packages, which are designed to deliver a specific set of abilities.
This approach enables extensibility, simplicity and easy maintenance..
@OrenMe - see if you are OK with my suggested changes - also, we already talk about modularity in the first part, so no point repeating it.

Reply:
so @dan-ziv, please use @AnnaAleksandrowicz suggestion